src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Colors [] colors = Colors.values();
+
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
It's bad practice to initialize dependencies in the method. You can initialize it in fields, but good practice initialization using a constructor.